### ISSUE TITLE
popup windows when Selecting Intro / Full Image with JCE is a little bit broken

### ISSUE THREAD

**Issue by woluweb** [Quantum-Manager/tracker#50]
**Describe the bug**
Hi there.
I refer to this screenshot :
https://twitter.com/woluweb/status/1324103266389659652

I have this issue on all my sites where I use Quantum, knowing that I have JCE Pro on all sites.
When clicking on the Image button within the JCE Editor, the popup is just fine.
But when clicking on the "Select Intro Image" or "Select Full Image" of an Article, then the popup is "broken" : there is some height issue so that you don't see the button (bc you have to scroll to see it) and typically end-users don't understand this.

By the way, txs a lot for this excellent extension :)

**Comment by dmitriitux** [comment]
@woluweb you can disable the system plugin from JCE then quantum will be used. If you need to leave JCE there, then I need to fix this error for JCE

**Comment by dmitriitux** [comment]
I'll see why there is a css conflict

**Comment by woluweb** [comment]
Txs @dmitriitux !
Yes, I know JCE could be disabled but it is not always possible (for example, all my sites have JCE but not all have Quantum. So that the documentation shows the Image selection with JCE. And then for people who want to get images from Unsplash, resize, crop etc I propose Quantum).

Indeed there is probably a little css conflict. If needed, I can provide you access to one of my websites.
Txs a lot for Quantum. It really rocks !

**Comment by dmitriitux** [comment]
@woluweb please send access to the site, I'll look there. thanks
email: cymbal@delo-design.ru

**Comment by dmitriitux** [comment]
@woluweb did you not send access? I didn't seem to receive a letter

**Comment by woluweb** [comment]
Hi @dmitriitux  !
Sorry I could not get back to you sooner.

Actually I have made further testing.
And I discovered the following (making fresh installs and comparing with all my current websites, which all have the issue) : it is actually a matter of Order : 
"System JCE" was before "System Quantum".
So when editing an article, when clicking on Intro Image or Full Image we had the screenshot I showed.

When reordering to put
System - Redirect Media (com_media) call to Quantum Manager
before
System - JCE
then there is no issue :)

So maybe something to mention in the Documentation of Quantum.

Txs again for your follow-up and -above all- for this excellent tool which is really appreciated by all the people who discovered it already.
